deploy/setup/setup.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Clear screen
+clear || printf "\033c"
```

Comment:
In my terminal this deletes the whole scroll back buffer. I would prefer the script not doing that. I think it is also fine to just start with printing into the terminal in the state it currently is.